tests/unit/test_charm.py

```diff
@@ -51,11 +52,11 @@ def test_charm_reaches_active_status(urlopen_mock):
     assert: the charm reaches ActiveStatus.
     """
     with open("tests/unit/files/metadata_1.xml", "rb") as metadata:
-        cm = MagicMock()
-        cm.getcode.return_value = 200
-        cm.read.return_value = metadata.read()
-        cm.__enter__.return_value = cm
-        urlopen_mock.return_value = cm
+        magic_mock = MagicMock()
```

Comment:
Should this describe what is being mocked rather than just its type?

Reply:
Indeed